test/configs/zombie_data_preservers_embeded_dancebox.json

```diff
@@ -125,26 +125,28 @@
                 {
                     "name": "DataPreserver-2000",
                     "validator": false,
-                    "command": "../target/release/container-chain-simple-node",
+                    "command": "./scripts/data-preservers-run-hack.sh ../target/release/container-chain-simple-node rpc-provider --",
```

Comment:
Why can't we just specify the exact command without args, it will execute it as is, from what I see [here](https://github.com/paritytech/zombienet/blob/main/javascript/packages/orchestrator/src/providers/native/nativeClient.ts#L488).

So, theoretically, we make args an empty array and put all the needed arguments to the command. Would it work?